src/main/java/com/exadel/discount/model/security/Permission.java

```diff
@@ -0,0 +1,13 @@
+package com.exadel.discount.model.security;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+public enum Permission {
+    VENDORS_ORDINARY_ACCESS("vendors:allUsers"),
+    VENDORS_PRIVILEGED_ACCESS("vendors:privilegedUsers");
+
```

Comment:
I was sure - security part is just for User Entity - to separate exactly users and admins, but vendors do not  have access to discount db at all, just through admins. Did the concept change?

Reply:
The answer was given in the Skype